components/logged_in/logged_in.jsx

```diff
@@ -178,12 +190,31 @@ export default class LoggedIn extends React.PureComponent {
         }
 
         switch (type) {
-        case 'user-activity-update':
+        case 'register-desktop': {
+            const {version} = message;
+            if (!window.desktop) {
+                window.desktop = {};
+            }
+            window.desktop.version = semver.valid(semver.coerce(version));
+            break;
+        }
+        case 'user-activity-update': {
+            const {userIsActive, manual} = message;
+
             // update the server with the users current away status
             if (userIsActive === true || userIsActive === false) {
                 WebSocketClient.userUpdateActiveStatus(userIsActive, manual);
             }
             break;
         }
+        case 'notification-clicked': {
```

Comment:
shouldn't this also mark user as being active?

Reply:
Good question! It's actually not necessary because when the user moves their mouse to click the notification, the Desktop app will detect that the user is no longer idle and will automatically call then `user-activity-update` case just above this to update the status.

That's a new v4.3 feature. 🙂